src/dash-playlist-loader.js

```diff
@@ -528,7 +518,7 @@ export default class DashPlaylistLoader extends EventTarget {
 
     // request the specified URL
     this.request = this.vhs_.xhr({
-      uri: this.srcUrl,
+      uri: this.masterPlaylistLoader_.srcUrl,
```

Comment:
Since we return early above for non master loaders, would it be clearer to use just `this`?

Reply:
I think we want it to be like this because:
1. The distinction will help us migrate master playlist loader code into a new file after these refactors.
2. It's much easier to know that you are accessing the correct object if you always access it the same way.

Reply:
Makes sense. Thanks!